include/envoy/network/transport_socket.h

```diff
@@ -133,7 +133,7 @@ class TransportSocket {
   /**
    * @return the const SSL connection data if this is an SSL connection, or nullptr if it is not.
```

Comment:
s/SSL/TLS/?

Reply:
I will correct the comments asap. Thanks